### PR TITLE
Build and Disribute The Package with Setuptoolsto package project

### PR DESCRIPTION
**Pull request by namvnn**
__Files added:__
  - `pyproject.toml`: the file that tells build tools (like pip 10+ and build) what system is used and what is required for building.
  - `setup.cfg`: Static metadata includes the configuration file for setuptools.
  - `MANIFEST.in`: The file supports to include static files to the package.

__Files changed:__
  - `README.md`: Add an additional instruction to install the package via PyPI
  - `__main__`: Fixed the argument to have the correct argument `subparser` instead of `subparsers`

Note: The PATCH version increases because of backwards-compatible bug fixes.

The package [speaker-verification](https://pypi.org/project/speaker-verification/) is hosted on [PyPI](https://pypi.org/)

Please install the package and test it in your machine, and comment feedback below for further improvements.

Ref issue #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +43,18 @@ python -m speaker_verification validate --id <ID_NUMBER> --audio-path <PATH/TO/F
...
+### _Install via PyPI_
+
+The package [speaker-verification](https://pypi.org/project/speaker-verification/) is hosted on [PyPI](https://pypi.org/)
+
+```
+pip install speaker-verification
+```
+
+### _Install from Github_
+
```
Comment (aidanjgriffiths) [inline README.md:57]: Could you add a commit to change L63 to the following:
```
export PYTHONPATH=${PYTHONPATH}:/path/to/speaker-verification
```
I didn't change this when I modified the CI unfortunately, but I think we can easily address it here.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,2 @@
+include  ./speaker_verification/SQL/*.db
+include ./speaker_verification/models/*
```
Comment (aidanjgriffiths) [inline MANIFEST.in]: Could we do this to be more specific?
```
include ./speaker_verification/models/*.h5
```

**Review by aidanjgriffiths — APPROVED** [review]
Hey @nnfunny, changes look good and work for me. Checkout the comments I've made as suggestions. Also why `v0.1.2` instead of `v0.1.1`?

File: MANIFEST.in (hunk 2 above)
Comment (namvnn) [reply MANIFEST.in]: Yes, we can.

**Comment by namvnn** [comment]
@aidanjgriffiths  I used v0.1.2 instead of v0.1.1 because I published the v0.1.1 for testing on the real PyPI, and tried to update it again but it failed, and the PyPI requires me to use different version.

**Comment by aidanjgriffiths** [comment]
LGTM!

**Review by aidanjgriffiths — APPROVED** [review]
LGTM! Feel free to merge.

**Pull request merged**